### PR TITLE
fix: Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Fix typos in code and comments.

Based on the email sent by @vintagewang

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by szepeviktor** [comment]
https://www.merriam-webster.com/dictionary/preempt

**Comment by superhx** [comment]
Can we avoid modifying the original Apache Kafka code to prevent future merge conflicts

**Comment by szepeviktor** [comment]
All right.
The thing is fro me the whole code base is just test, I do not understand what it does.
Where is the "original Apache Kafka code"?

**Comment by superhx** [comment]
> All right. The thing is for me the whole code base is just text, I do not understand what it does. Where is the "original Apache Kafka code"?

A straightforward rule: It's okay to modify the file if the copyright is AutoMQ.

**Comment by szepeviktor** [comment]
Done 🍏 

54 files -> 12 files

File: metadata/src/main/java/org/apache/kafka/controller/stream/TopicDeletionManager.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public void cleanupDeletedTopics() {
                 try {
                     return this.cleanupDeletedTopics0();
                 } catch (Throwable ex) {
-                    LOGGER.error("[CLEANUP_DELETED_TOPIC],[UNEXPECT]", ex);
+                    LOGGER.error("[CLEANUP_DELETED_TOPIC],[UNEXPECTED]", ex);
```
Comment (szepeviktor) [inline metadata/src/main/java/org/apache/kafka/controller/stream/TopicDeletionManager.java:73]: Not sure here.
Please help me.

**Review by superhx — APPROVED** [review]

**Pull request merged**